tools-core/src/test/java/com/hedera/hashgraph/client/core/fileservices/LocalFileServiceAdapterTest.java

```diff
@@ -62,7 +62,7 @@ void listFiles() throws HederaClientException {
 	void listFiles_badFile() {
 		assertThrows(HederaClientException.class, () -> {
 			final var localFileServiceAdapter = new LocalFileServiceAdapter(TEST_PATH + "genesis.pem");
-			final var x = localFileServiceAdapter.listFiles();
+			@SuppressWarnings("unused") final var x = localFileServiceAdapter.listFiles();
```

Comment:
Instead of `@SuppressWarnings`, perhaps we just call `localFileServiceAdapter.listFiles()` without the assignment. And with that having said, do we need the return value of `localFileServiceAdapter.listFiles()` anywhere else? If not, change the function to a `void` instead.

Reply:
I removed the local variable. In the next pass I need to review this Interface and remove anything that is not needed, but that will probably be too big a change for this PR